scp/scp.cpp

```diff
@@ -45,6 +45,7 @@ void scpModule::Initialize(ScMemoryContext * context)
 {
   std::cout << "SCP START" << std::endl;
   ScAgentContext agentContext;
+  Keynodes::InitializeDeprecatedKeynodes();
```

Comment:
keynodes initialization was just fixed, but you've brought initialization back
maybe use something similar to solution used in any operator interpreter, like initialization on demand in get method without user knowing that there is some initialization going on